trigger-actions-framework/main/default/classes/TriggerActionFlowAddError.cls

```diff
@@ -16,7 +16,8 @@
 
 public inherited sharing class TriggerActionFlowAddError {
 	@TestVisible
-	private static final String CLASS_NAME = 'TriggerActionFlowAddError';
+	private static final String CLASS_NAME = TriggerActionFlowAddError.class
```

Comment:
Thank you for the updates using `Xyz.class.getName()` across all of these files. This is a very welcomed update!

Reply:
But of course!